docs/README.md

```diff
@@ -0,0 +1,43 @@
+## Kubernetes-Mesos Documentation
+
+This project combines concepts and technologies from two already-complex projects: Mesos and Kubernetes.
+It may help to familiarize yourself with the basics of each project before reading on:
+
+* [Mesos Documentation][1]
+* [Kubernetes Documentation][2] and [Key Concepts][4]
+  * [Labels][6]
+  * [Nodes][9]
+  * [Pods][7]
+  * [Replication Controller][8]
+  * [Services][5]
+
+It may also be helpful to understand the [known issues][3] with respect to this project.
+
+### Architecture
+
+A Mesos cluster consists of one or more masters, and one or more slaves.
+Kubernetes-Mesos (k8sm) operates as a Mesos framework that runs on the cluster.
+As a framework, k8sm provides scheduler and executor components, both of which are hybrids of Kubernetes and Mesos:
+the scheduler component integrates the Kubernetes scheduling API and the Mesos scheduler runtime, whereas;
+the executor component integrates Kubernetes kubelet services and the Mesos executor runtime.
+
+Multiple Mesos masters are typically configured to coordinate leadership election via Zookeeper.
```

Comment:
I added a link to this

On Tue, Jan 27, 2015 at 2:27 AM, adam-mesos <notifications@github.com>
wrote:

> In docs/README.md
> <https://github.com/mesosphere/kubernetes-mesos/pull/134#discussion_r23591013>
> :
>
> > +  * [Nodes][9]
> > +  * [Pods][7]
> > +  * [Replication Controller][8]
> > +  * [Services][5]
> > +
> > +It may also be helpful to understand the [known issues][3] with respect to this project.
> > +
> > +### Architecture
> > +
> > +A Mesos cluster consists of one or more masters, and one or more slaves.
> > +Kubernetes-Mesos (k8sm) operates as a Mesos framework that runs on the cluster.
> > +As a framework, k8sm provides scheduler and executor components, both of which are hybrids of Kubernetes and Mesos:
> > +the scheduler component integrates the Kubernetes scheduling API and the Mesos scheduler runtime, whereas;
> > +the executor component integrates Kubernetes kubelet services and the Mesos executor runtime.
> > +
> > +Multiple Mesos masters are typically configured to coordinate leadership election via Zookeeper.
>
> Etcd-based leader election is on the way; I've seen a live demo. More
> details should follow in https://issues.apache.org/jira/browse/MESOS-1806
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/kubernetes-mesos/pull/134/files#r23591013>.
>



-- 
James DeFelice
585.241.9488 (voice)
650.649.6071 (fax)